brute-force-connect/build.gradle.kts

```diff
@@ -1,15 +1,28 @@
 description = "Kafka Connect Converter that deserializes messages of an unknown serialization format"
 
+repositories {
+    // required for kafka-streams-json-schema-serde dependency
+    maven(url = "https://jitpack.io")
```

Comment:
Really? Then please update README line 31

Reply:
Yes: https://github.com/confluentinc/schema-registry/issues/2171

`kafka-streams-json-schema-serde` is just a test dependency, hough, therefore I don't see the need to include it in the ReadMe. If someone uses the json-serde, I think they'll know.